tests/php/sync/test_class.jetpack-sync-base.php

```diff
@@ -10,7 +10,7 @@
 use Automattic\Jetpack\Sync\Server;
 use Automattic\Jetpack\Sync\Modules\Posts;
 
-Main::init();
+Jetpack::configure();
```

Comment:
It looks like the `plugins_loaded` action is already done when this file is loaded, so the Config methods that are hooked to `plugins_loaded` aren't executed.

Maybe calling `Jetpack::configure()` in the bootstrap.php file will fix some of the test errors and failures. I ran the tests locally, and many of the errors and failures were resolved. The `_manually_load_plugin()` function in bootstrap.php is hooked to the `plugins_loaded` action with a priority of 1. We could call `Jetpack::configure()` after the [`require jetpack.php` statement in this method](https://github.com/Automattic/jetpack/blob/master/tests/php/bootstrap.php#L73).

Reply:
That did it, thanks a lot for this suggestion!